benchbuild/source/git.py

```diff
@@ -0,0 +1,112 @@
+"""
+Declare a git source.
+"""
+from typing import Any, List
+
+import attr
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'attr'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)